android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java

```diff
@@ -287,6 +288,12 @@ public void setShowsVerticalScrollIndicator(WebView view, boolean enabled) {
     view.setVerticalScrollBarEnabled(enabled);
   }
 
+  @ReactProp(name = "downloadMessage")
```

Comment:
Do you think `downloadingMessage` might be clearer, just to avoid any possible confusing about it being a "download success" message...?

(I know the original variable name was `downloadMessage`, but since that was just a local variable in the function there was less likely to be any confusion)

Reply:
changed to downloadingMessage